CMakeLists.txt

```diff
@@ -426,9 +426,9 @@ add_subdirectory("docs/doxygen")
 file(GLOB_RECURSE ALL_SOURCE_FILES RELATIVE ${CMAKE_SOURCE_DIR}
     src/*.cc src/*.c src/*.h)
 foreach(FILE ${ALL_SOURCE_FILES})
-    string (FIND ${FILE} "/isotp_conn/" EXCLUDE_DIR_FOUND)
+    string (FIND ${FILE} "/libaktualizr-isotpconn/" EXCLUDE_DIR_FOUND)
```

Comment:
This should be `libaktualizr-socketcan` in order to match the actual directory, right?

But on second thought, should the subdirectory actually be `libaktualizr-isotp`? I think that's the more specific, higher-level protocol being used here, and this is code is less specific to socketcan. But I'm open to debate on that.

Reply:
Yes, libaktualizr-isotp is more close to the name of our source file, I changed it.